codespell_lib/data/dictionary.txt

```diff
@@ -52588,6 +52588,7 @@ sempaphores->semaphores
 semphore->semaphore
 semphores->semaphores
 sempphore->semaphore
+sems->seems, stems, semis, sens,
```

Comment:
Alright, that makes sense. Just one more suggestion.
```suggestion
sems->seems, sees, stems, semis, sens,
```

Reply:
Nice, thanks a lot for the additional suggestion :+1:

Reply:
And perhaps `seams` as well.

Reply:
Wow, lots of words sems could be a typo of.

Reply:
Added as well, thanks a lot again for the suggestions :+1: